app/src/main/res/layout/sign_up_fragment.xml

```diff
@@ -11,113 +11,117 @@
             type="org.fossasia.openevent.app.common.data.models.User" />
     </data>
 
-    <android.support.v4.widget.NestedScrollView
+    <FrameLayout
```

Comment:
Still that is better because this method is wrapping it inside an unnecessary hierarchy, which is also bad for performance